### PR TITLE
Feature/podcasts detail

### PR DESCRIPTION
**Pull request by jdwitten**
* Gets us started playing podcasts, just starts playing when you navigate to the page and ends when you leave.
* Need to add play/pause controls

### CODE REVIEW

File: iOS Client/PodcastsController/PodcastDetailViewController.swift [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        // Do any additional setup after loading the view.
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        MBStore.sharedStore.subscribe(self)
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillAppear(animated)
```
Comment (AlexRamey) [inline iOS Client/PodcastsController/PodcastDetailViewController.swift]: viewWillDisappear?

File: iOS Client/HTTPClient/MBClient.swift [hunk 2]
```diff
@@ -127,6 +127,23 @@ class MBClient: NSObject {
         }.resume()
     }
     
+    func getPodcastAudio(for podcast: MBPodcast, completion: @escaping (Data?, Error?) -> Void ) {
```
Comment (AlexRamey) [inline iOS Client/HTTPClient/MBClient.swift]: It seems like the AVPlayer handles this for us down below. Is this function used?

File: iOS Client/PodcastsController/PodcastDetailViewController.swift [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        MBStore.sharedStore.unsubscribe(self)
+    }
+    
+    func configureBackButton() {
+        self.navigationItem.leftBarButtonItem = UIBarButtonItem(title: "Back", style: .done, target: self, action: #selector(self.backToPodcasts(sender:)))
+    }
+    
+    @objc func backToPodcasts(sender: AnyObject) {
+        player = nil
```
Comment (AlexRamey) [inline iOS Client/PodcastsController/PodcastDetailViewController.swift]: Why set player to nil? Won't the PopCurrentNavigation action deallocate this view controller and, in turn, its fields? Is there a noticeable delay prompting this?

**Review by AlexRamey — CHANGES_REQUESTED** [review]
basically seems fine, just a few comments.

File: iOS Client/HTTPClient/MBClient.swift (hunk 2 above)
Comment (jdwitten) [reply iOS Client/HTTPClient/MBClient.swift]: I had this before I realized I could just pass a url. I can remove

File: iOS Client/PodcastsController/PodcastDetailViewController.swift (hunk 1 above)
Comment (jdwitten) [reply iOS Client/PodcastsController/PodcastDetailViewController.swift]: nice catch

File: iOS Client/PodcastsController/PodcastDetailViewController.swift (hunk 3 above)
Comment (jdwitten) [reply iOS Client/PodcastsController/PodcastDetailViewController.swift]: yeah probably don't need to do this at all

**Review by AlexRamey — APPROVED** [review]

**Pull request merged**